components/style/properties.mako.rs

```diff
@@ -94,8 +94,10 @@ class Method(object):
 
 class StyleStruct(object):
     def __init__(self, name, inherited, gecko_name, additional_methods):
-        self.name = name
-        self.ident = to_rust_ident(name.lower())
+        self.struct_name = "Servo" + name
+        self.trait_name = name
+        self.trait_name_lower = name.lower()
```

Comment:
I deliberately killed the `name` property here, to get runtime errors from the Python/Mako code, just to have to make an explicit decision on each occasion as to whether the trait or struct name should be used in the code generation.